web/src/js/gcping.js

```diff
@@ -17,11 +17,11 @@
 // TODO: Show regions on a map, with lines overlayed according to ping times.
 // TODO: Add an option to contribute times and JS geolocation info to a public BigQuery dataset.
 
-const GLOBAL_REGION_KEY = "global",
-  PING_TEST_RUNNING_STATUS = "running",
-  PING_TEST_STOPPED_STATUS = "stopped",
-  INITIAL_ITERATIONS = 10,
-  btnCtrl = document.getElementById('stopstart');
+const GLOBAL_REGION_KEY = "global";
+const PING_TEST_RUNNING_STATUS = "running";
+const PING_TEST_STOPPED_STATUS = "stopped";
+const INITIAL_ITERATIONS = 10;
+const btnCtrl = document.getElementById("stopstart");
```

Comment:
Does this seem better to everyone? I wasn't sure of this in the original PR too. But if everyone prefers this style over a coma separated declaration , then we're good :)

Reply:
One of the nice things about prettier is we don't have to think about formatting. We just accept whatever it gives us and move on to other things. So if prettier does this, I like it. :-)

Reply:
Yeah. I have no care which way it is, just that we don't ever talk about the way it is :p